### PR TITLE
Build Out Organizations Edit UI

### PR DESCRIPTION
**Pull request by ncbui**
## What type of PR is this? (check all applicable)

- [] Feature
- [] Refactor
- [] Bug Fix
- [] Optimization
- [] Documentation Update
- [x] Other (describe: UI styling)

## Description of what PR does
Completes step 1 of #12 by styling the view at http://127.0.0.1:3000/organizations/:id/edit. 

To build for easier edits in the future, I've used an instance method for the grid layout. In the org controller, I created and defined instances for @form_label, @form_text_fields, and @form_submit_button classes. In the edit view, I referenced those instances. Changes to the instances in the controller will be reflected in the view. 

Note: This PR does not address the layout changes that happen when an error pops up. See #10 Step 4 and [slack thread](https://ourtimefortech.slack.com/archives/G01NBP423T9/p1618427794004600) for details.

## Related PRs and/or Issues (if any)
-
-

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed _(please include reasons for why tests aren't needed)_
- [ ] no, because I need help
- [x] no, they will be added later (please create an Issue for it)

## Added to documentation?

- [ ] Yes, project README
- [ ] No documentation needed

### CODE REVIEW

File: app/assets/stylesheets/organizations.scss [hunk 1]
```diff
@@ -1,3 +1,21 @@
 // Place all the styles related to the Organizations controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: https://sass-lang.com/
+$form-padding: 40px;
+$form-margin: 20px 0;
+$field-padding: 10px 0;
```
Comment (saradotlog) [inline app/assets/stylesheets/organizations.scss:6]: Hey @ncbui! Wanted to say, maybe it is better to have a _variables.scss file where we add all the variables? So in that way they can be used across all the sass files, as those padding and margins will be probably used in other views as well?

File: app/controllers/organizations_controller.rb [hunk 2]
```diff
@@ -26,6 +28,9 @@ def create
   end
 
   def edit
+    @form_label = "col-12 col-md-4 col-lg-2"
+    @form_text_fields = "col-12 col-md-8 col-lg-10 mx-auto"
```
Comment (saradotlog) [inline app/controllers/organizations_controller.rb:32]: I also have the same doubt about this... If they are going to be used in multiple places and we want to define them, wouldn't it be better to have them in a shared/global file?

Comment (saradotlog) [reply app/controllers/organizations_controller.rb:32]: Also, one thing I noticed while reviewing, is that if there is an error in the form all the bootstrap classes for all the form disappear, while if you add those classes in the label/text_field (`<%= f.label :org_address, :class => "col-12 col-md-4 col-lg-2" %>
                    <%= f.text_field :org_address, :class => "col-12 col-md-8 col-lg-10 mx-auto" %>`) It is fine after the validation.
In my opinion I prefer to have the classes in the view, not defined in the controller. I find it more intuitive and easy to find/change/maintain. But of course I adapt to what all of you prefer

File: app/assets/stylesheets/organizations.scss (hunk 1 above)
Comment (ncbui) [reply app/assets/stylesheets/organizations.scss:6]: Hey @saradotlog , that is a good point for variables like this. Let's discuss it at the stand up. For this PR specifically, this suggestion is beyond it's scope. I have not taken the time to test this out on all other resources and cannot presume to say that these specific margins and padding will work for everyone.

File: app/controllers/organizations_controller.rb (hunk 2 above)
Comment (ncbui) [reply app/controllers/organizations_controller.rb:32]: Hey @saradotlog , same sentiment about this one. I have not worked on other resources and cannot presume that this grid layout will work best for them. We discussed building something simple in anticipation of it being edited and changed once we have a better picture of our product. I agree that some global style guides will be useful, let's discuss it more at the meeting

Comment (ncbui) [reply app/controllers/organizations_controller.rb:32]: Also, the video I shared was from when the code had the classes inside the label/text field. The warping still occurs before a variable pattern was added.

File: app/assets/stylesheets/organizations.scss (hunk 1 above)
Comment (saradotlog) [reply app/assets/stylesheets/organizations.scss:6]: Yeah, it is, no point in solving it here. Just saying that maybe we should add that variables file before we all start adding variables that might do the same. It is luckily that the edit forms have the same kind of styling, for orgs, workers and even shifts, so the styles might be shared. It's true that we might need to do a global overview

Comment (ncbui) [reply app/assets/stylesheets/organizations.scss:6]: Thanks for reviewing this @saradotlog, I'm going to refactor this to use more of bootstrap's utility classes.

File: app/controllers/organizations_controller.rb (hunk 2 above)
Comment (saradotlog) [reply app/controllers/organizations_controller.rb:32]: So I fetched the new code and tested to edit the form again.
This is what I meant about the variables: 
When loading the first time "edit organisation":
View1
Everything looks correct with the bootstrap grid classes added.
After it, I try to update the form, leaving one field empty to force an error. The field-with-errors appears in the field I left empty, so the grid breaks. But also the grid classes that are supposed to be in the fields don't appear in the html:
View2
Instead if I add the classes directly in the edit.html.erb file like this 
```
<div class="row organizations-field py-2">
        <%= f.label :org_name, :class => "col-12 col-md-4 col-lg-2" %>
        <%= f.text_field :org_name, :class => "col-12 col-md-8 col-lg-10 mx-auto" %>
</div>
```
and try to edit the form again, the only classes that seems to appear are the ones added in the view, not the ones in the controller:
view3

Maybe the form validations doesn't go through the controller again and those variables are empty when reloading? 
Anyway, as I said, I'm open to everything, but I find much easier and intuitive to add the classes in the appropriate view.

Comment (Timilehin08) [reply app/controllers/organizations_controller.rb:32]: Oh oh now i get what you were saying yesterday @saradotlog

Comment (ncbui) [reply app/controllers/organizations_controller.rb]: Thanks for these screenshots @saradotlog ! Have moved the classes to the view file instead

**Review by Timilehin08 — COMMENTED** [review]
Welldone! @ncbui The changes look good to me checking them out locally.

**Comment by saradotlog** [comment]
Thanks @ncbui ! LGTM now! great job! 😃

**Review by msarit — APPROVED** [review]
Awesome job!

**Pull request merged**